### PR TITLE
New ioctl command for alias import and unit test

### PR DESCRIPTION
**Pull request by RuneAkko**
#1768

### CODE REVIEW

File: ioctl/newcmd/alias/alias.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2019 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package alias
```
Comment (dustinxie) [inline ioctl/newcmd/alias/alias.go:7]: can you introduce the background and purpose of this PR?
it looks like the code is duplicate of existing alias.go

Comment (RuneAkko) [reply ioctl/newcmd/alias/alias.go:7]: please reference issue#1768, and the alias.go in ioctl\newcmd is used for the same reason of the bc.go in ioctl\newcmd. This PR is about migrating old ioctl cmd to newcmd.

File: ioctl/newcmd/alias/alias.go [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+	ErrNoAliasFound = errors.New("no alias is found")
+)
+
+// Flags
+var (
+	format      string
+	forceImport bool
+)
+
+//type alias struct {
```
Comment (raullenchai) [inline ioctl/newcmd/alias/alias.go:46]: remove if not needed

File: ioctl/newcmd/alias/alias.go (hunk 1 above)
Comment (raullenchai) [reply ioctl/newcmd/alias/alias.go:7]: should remove the old alias.go?

File: ioctl/newcmd/alias/aliasimport_test.go [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+	client := mock_ioctlclient.NewMockClient(ctrl)
+	client.EXPECT().SelectTranslation(gomock.Any()).Return("mockTranslation", config.English).Times(4)
+
+	cmd := NewAliasImportCmd(client)
+
+	result, err := util.ExecuteCmd(cmd, `{"name":"a","address":"io1uwnr55vqmhf3xeg5phgurlyl702af6eju542sx"}`)
+
+	require.NoError(t, err)
+	require.NotNil(t, result)
+
```
Comment (CoderZhi) [inline ioctl/newcmd/alias/aliasimport_test.go:26]: Add failure case please.

**Pull request closed without merging**